apps/web/utils/string.tsx

```diff
@@ -53,6 +72,24 @@ export const parseStringToComponents = (
               }
             }
             break;
+          case pdfDownloadLinkName:
+            {
+              const PDFDownloadLinkComponent =
+                customComponents[pdfDownloadLinkName];
+              const props = attributesToProps(domNode.attribs);
+
+              if (PDFDownloadLinkComponent) {
```

Comment:
What scenario would this be false?

Reply:
Good call! Not needed anymore since I refactored the type for the `customComponents` object. Removed!